gui/packages/desktop/src/main/notification-controller.js

```diff
@@ -115,22 +107,14 @@ export default class NotificationController {
     }
   }
 
-  _closeNotification(notification: Notification) {
-    // If the notification is closed too soon, it might still get shown. If that happens, close()
-    // should be called again so that it is closed immediately.
-    // Tracking issue: https://github.com/electron/electron/issues/12887
-    notification.addEventListener('show', () => {
-      notification.close();
-    });
+  _scheduleNotification(notification: Notification) {
+    this._addPendingNotification(notification);
 
-    notification.close();
+    notification.show();
   }
 
   _addPendingNotification(notification: Notification) {
-    // Quirk: chromium postpones the 'close' event until new notifications pump the queue or window
```

Comment:
Would be awesome to test that this now does not require the same hack as with HTML5 APIs.